filetests/parser/tiny.cton

```diff
@@ -158,9 +158,12 @@ ebb0(v1: i32):
     v6 = load.i64 aligned notrap v1
     v7 = load.i64 v1-12
     v8 = load.i64 notrap v1+0x1_0000
+    v9 = load_complex.i64 v1+v2
+    v10 = load_complex.i64 v1+v2+0x1
```

Comment:
In the current patch, is it valid to have a load_complex with just one register? If so, we should test that too. But if not, it's not important (for now).

Reply:
I didn't implement single value load_complex in this patch, unfortunately.